### ISSUE TITLE
Toy Dynamics should have separate System object

### ISSUE THREAD

**Issue by dwhswenson** [openpathsampling/openpathsampling#73]
Per discussion in https://github.com/choderalab/opentis/issues/65#issuecomment-62613931 ff., Toy Dynamics should have a separate object (perhaps called `ToySystem` or `ToySystemState`), such that for `simulation = ToySimulation()`, instead of

``` python
vel = simulation.velocities
pos = simulation.positions
```

we have

``` python
vel = simulation.current_system.velocities
pos = simulation.current_system.positions
```

This makes Toy Dynamics more pedagogically useful

**Comment by jhprinz** [comment]
What is the status on this. Do we still need to make changes here?

**Comment by dwhswenson** [comment]
Milestoned future. It should be done, but not in 1.0.

**Comment by jhprinz** [comment]
I think we can close this as it is replaced by using features and the new Snapshot factory.

**Comment by dwhswenson** [comment]
> I think we can close this as it is replaced by using features and the new Snapshot factory.

No, that misses the intent. See https://github.com/choderalab/openpathsampling/issues/73#issuecomment-63059732.

**Comment by jhprinz** [comment]
Thanks for reminding me.
